test/test-utils/catchAsyncError.ts

```diff
@@ -0,0 +1,9 @@
+const catchAsyncError = (done, cb) => {
```

Comment:
There is also a `wrap` in the test-utils that does something very similar. Could we merge these into one single method? Perhaps an idea for a follow-up PR?

Reply:
We could merge - I just extracted because it wasn't necessary here.

Reply:
Great. Maybe an idea to do it in a follow-up PR to keep the code clean and prevent duplication of methods from appearing.